src/cli/index.js

```diff
@@ -170,6 +170,7 @@ export function main({
   const run = (): Promise<void> => {
     invariant(command, 'missing command');
     return command.run(config, reporter, commander, commander.args).then(exitCode => {
+      reporter.close();
```

Comment:
the reporter stuff is from the revert of the commit 280b6eb59ac6a8aaafdde1d86b48372ac76149f4 :(  it was necessary to have the feature working after rebasing, but let me re-revert it once a fix lands.